### ISSUE TITLE
Access to website

### ISSUE THREAD

**Issue by michelechambers** [dask/community#202]
Hello - 

I'd like to get access to the dask.org website and Google Analytics for our digital marketing manager (Lily Melnyk (lily@coiled.io) so that we can help with SEO strategies to increase traffic and discoverability of content that's on dask.org. 

Please let me know if you have any questions.
Thanks,
Michele
michele@coiled.io

**Comment by quasiben** [comment]
I don't have any objections.  At a Dask Monthly meeting or some other venue could we ask Lily to present on their findings and/or strategy ?

**Comment by michelechambers** [comment]
Absolutely. That's a great way to get everyone's feedback so we can iterate on improvements faster.

**Comment by jacobtomlinson** [comment]
Having someone like Lily engaged in the work that @jsignell and I have been doing on the docs would also be super useful.

**Comment by LilyMelnyk** [comment]
Popping in to say hello 👋 

Looking forward to working with folks!

**Comment by mrocklin** [comment]
Heads up, I'm also granting @ian-r-rose access.  I suspect that folks here don't object to that, but please let me know if they do.
